backend/handlers/handlers.go

```diff
@@ -681,6 +682,28 @@ func (handlers *Handlers) postElectrumCheckHandler(r *http.Request) (interface{}
 	}, nil
 }
 
+func (handlers *Handlers) postSocksProxyCheck(r *http.Request) (interface{}, error) {
+	var endpoint string
+	if err := json.NewDecoder(r.Body).Decode(&endpoint); err != nil {
+		return nil, errp.WithStack(err)
+	}
+
+	type response struct {
+		Success      bool   `json:"success"`
+		ErrorMessage string `json:"errorMessage,omitempty"`
```

Comment:
Rename to just `message` to be consistent with some existing API responses? See the other thread with @thisconnect

Reply:
Other handlers also use this, so changing this wouldn't make it more consistent.